docs/getting_started.txt

```diff
@@ -30,15 +30,15 @@ Usage
 
 There are a number of ways you can use ``crispy-tailwind``. The simplest way
 is to use the ``|crispy`` filter. This will add an opinionated `Tailwind` style
-to your form. If you require more control you can use the ``crispy`` tags and
-utilise the full power of the ``crispy-forms`` through it's ``FormHelper`` and
+to your form. If you require more control, you can use the ``crispy`` tags and
+utilise the full power of the ``crispy-forms`` through its ``FormHelper`` and
 ``Layout`` capabilities.
 
-`|crispy` filter
-----------------
+``|crispy`` filter
+------------------
 
-The crispy filter is the simpler way to add `Tailwind` style to your form. It
-avoids the complexity of ``FormHelper`` and it comes with an opinionated form
+The crispy filter is the simplest way to add `Tailwind` style to your form. It
```

Comment:
Any other options to avoid "simple" or "easy" here?  

I think the `crispy-form` docs go with something like _the original and still useful way_ :thinking:

Reply:
I think it's important to communicate why someone would use one method over another, and how much effort is required would seem to be the most relevant difference in this case. While I don't think there's any problem using "simple" in this particular context, I just pushed an updated commit in which I changed the wording to "the method requiring the least amount of effort", which I think accurately conveys how this method is different from the other options. That being the case, how do things look now?

Reply:
I'm probably being overly pedantic on second review. The reasons you articulated is probably why I went with _simpler_ in the first case. Anyway, I like your amends so let's have this.